velox/dwio/common/ScanSpec.h

```diff
@@ -255,6 +255,12 @@ class ScanSpec {
   // This may change as a result of runtime adaptation.
   bool hasFilter() const;
 
+  /// Assume this field is read as null constant vector (usually due to missing
+  /// field), check if any filter in the struct subtree would make the whole
+  /// vector to be filtered out.  Return false when the whole vector should be
```

Comment:
There are double space before 'Return'

Reply:
That's intentional to have double space between sentenses in monospace font